.github/workflows/api-test.yml

```diff
@@ -90,6 +90,12 @@ jobs:
         case:
           - name: Tenant
             class: org.apache.dolphinscheduler.api.test.cases.TenantAPITest
+          - name: Project
+            class: org.apache.dolphinscheduler.api.test.cases.ProjectAPITest
+          - name: WorkFlow
+              class: org.apache.dolphinscheduler.api.test.cases.WorkFlowAPITest
+          - name: WorkFlowInstance
+              class: org.apache.dolphinscheduler.api.test.cases.WorkFlowInstanceAPITest
```

Comment:
I think we should combine workflow use cases of the same task type into the same file since workflow operation and query belong to the same type. And it's better rename to `WorkFlowShellAPITest`.

Reply:
Thank you for your reminder. Corresponding modifications have been made here